README.md

```diff
@@ -1,2 +1,71 @@
 # M24SR64-Y
 Arduino library to support the dynamic NFC/RFID Tag IC dual interface M24SR64-Y
+
+## Important note
+
+Before using this library, install the ArduinoSTL library.  
+Make sure to include the patch number with commit
```

Comment:
Seems difficult to apply for non github user.

Reply:
I agree but we have no choice.
@cparata push a PR but we are waiting the merge.

Reply:
Please, use this PR to test: https://github.com/stm32duino/Arduino_Core_STM32/pull/129
This allow to use STL without Arduino STL.
Build ok by removing all  ```ArduinoSTL.h``` include in NFC sketches.